weblate/settings_docker.py

```diff
@@ -1301,6 +1301,7 @@
 
 # Avatars
 ENABLE_AVATARS = get_env_bool("WEBLATE_ENABLE_AVATARS", True)
+AVATAR_URL_PREFIX = os.environ.get("WEBLATE_AVATAR_URL_PREFIX", "")
```

Comment:
It should keep the default value:

```suggestion
AVATAR_URL_PREFIX = os.environ.get("WEBLATE_AVATAR_URL_PREFIX", "https://www.gravatar.com/")
```

Reply:
Understood, I pushed those changes, the correct default value is now used.